test/src/main/java/org/apache/accumulo/test/functional/FateConcurrencyIT.java

```diff
@@ -220,7 +225,15 @@ public void getFateStatus() {
 
     Future<?> compactTask = startCompactTask();
 
-    assertTrue("compaction fate transaction exits", findFate(tableName));
+    try {
+
+      assertTrue("compaction fate transaction exits", findFate(tableName));
```

Comment:
The blockUntilCompactionsRunning() essentially does this test.  Perhaps that should return a boolean as to whether a compaction was found and that should be tested instead of depending on the timing of the compaction start on 226.

Reply:
I'll incorporate both suggestions and push changes in near future.